src/menu.ts

```diff
@@ -766,6 +769,13 @@ export class Menu<C extends Context = Context> extends MenuRange<C>
             ctx,
             async (btn, i, j): Promise<InlineKeyboardButton> => {
                 const text = await uniform(ctx, btn.text);
+                btn.url = btn.url && (await uniform(ctx, btn.url))
+                btn.switch_inline_query =
+                    btn.switch_inline_query &&
+                    (await uniform(ctx, btn.switch_inline_query_current_chat))
+                btn.switch_inline_query_current_chat =
+                    btn.switch_inline_query_current_chat &&
+                    (await uniform(ctx, btn.switch_inline_query_current_chat))
```

Comment:
You are setting these values on the object even when the properties are absent. This is undesired. Have a look at how middleware is handled, and replicate this pattern:
```ts
if("url" in btn") {
  // ...
}
```